assets/src/scss/components/_share-buttons.scss

```diff
@@ -3,62 +3,57 @@
 
   .share-btn {
     border: none;
-    color: $white;
     font-size: $font-size-md;
     opacity: 0.9;
-    width: 4rem;
+    width: 3rem;
     text-align: center;
     line-height: 2.7rem;
-    height: 2.7rem;
-    border-radius: 4px;
+    height: 3rem;
+    border-radius: 50%;
     margin-bottom: 0;
+    background: $share-buttons-bg-color;
```

Comment:
For example,
`background: var(--color-background-button_share--secondary-passive);`